src/FieldExchanger.jl

```diff
@@ -209,14 +208,8 @@ Updates the surface component model cache with the current coupler fields beside
 - `csf`: [NamedTuple] containing coupler fields.
 """
 function update_sim!(sim::Interfacer.SurfaceModelSimulation, csf, area_fraction)
```

Comment:
We don't need `area_fraction` here anymore?

Reply:
I don't think so. This is the case we talked about the other day where we get the radiative flux computed by the atmosphere and provide it to the surface models in `csf.F_radiative`. We decided we should pass the unweighted flux to each surface